randomizer/Lists/DoorLocations.py

```diff
@@ -739,7 +735,7 @@ def GetBossLobbyRegionIdForRegion(region_id, region):
             kong_lst=[Kongs.donkey],
             group=2,
             moveless=False,
-            logic=lambda l: (l.isdonkey and l.strongKong) or l.phasewalk or l.generalclips,
+            logic=lambda l: l.isdonkey and l.strongKong,
```

Comment:
This door location is halfway the quicksand tunnel. Glitches won't save you from getting your feet hot. (as far as I have tried, you cannot talk to wrinkly from within the wall)